### DISCUSSION TITLE
Method to execute 'adb shell' commands in webdriverIo

### DISCUSSION THREAD

**Discussion by Kiran2205** [webdriverio/webdriverio#12101] [Community]
Hello wonderful team,

Can any one please help me, on how I can execute 'adb' command or script in webdriverIO. I am trying to start and login to the app directly with intent or using deep link.

Thanks in advance

Regards,
Kiran

**Comment by christian-bromann** [comment]
This should be possible via:

```ts
driver.executeScript("mobile: shell", {"command": "rm", "args": ["-rf", "/mnt/sdcard/Pictures/*.*"]});
```

Read more on this: https://www.headspin.io/blog/running-arbitrary-adb-commands-via-appium
